### DISCUSSION TITLE
Shelf Mushroom

### DISCUSSION THREAD

**Discussion by Soro335** [FrozenBlock/WilderWild#22] [Ideas]
I think we need to finish the birch forest as soon as possible and make a public release of the mod, so we need a shelf mushroom.

This block should work as sea pickles/candles and also be place as an amethyst cluster/glow lichen.
 
I tried to add it today but it's too complicated for me so hopefully others can add it.

**Comment by Soro335** [comment]
@AViewFromTheTop I hope you can help with this

**Comment by AViewFromTheTop** [comment]
wth is a discussion
